Demo/Demo/DemoMerchantAPI.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+
+final class DemoMerchantAPI {
+
+    static let sharedService = DemoMerchantAPI()
+
+    private init() {}
+
+    func createOrder(orderParams: CreateOrderParams, completion: @escaping ((Result<Order, URLResponseError>) -> Void)) {
+        // TODO: get environment from settings in Demo app
+        guard let url = URL(string: DemoSettings.Environment.sandbox.baseURL + "/order?countryCode=US") else {
```

Comment:
can we turn this into a function 
`func url(endpoint: String, environment: DemoSettings.Environment) -> URL?`
or `func url(endpoint: String) -> URL?` if we have an instance variable `environment`
so we don't have to combine base url string everytime

Reply:
Hm, not sure I understand what benefit that will really give us. The URL will be different for different request types, creatOrder, processOrder, etc.

Reply:
I think if we have that function, here we'll do this:
```
guard let url = url(endpoint: "/order?countryCode=US") else ...
```
that way this function and `func processOrder()` don't need to care what the baseURL is and do the string combining anymore, func `url(endpoint: String)` will handle it for us.
Also if in the future we wanna change/rename `DemoSettings.Environment.sandbox.baseURL`, we'll just need to make one change in `func url(endpoint: String)` instead of having to change in multiple functions (`createOrder()`, `processOrder()`, etc.)

Reply:
but also this is a very minor change request, totally cool if we don't do this :smile:

Reply:
Jax & I tried this out. It doesn't really make the code more readable or much more DRY, since it's one line of code that is pretty simple. We opted to leave as-is.

Reply:
ah gotcha! Yep then everything looks good to me!!! :smile: